support/cas-server-support-pac4j-core-clients/src/main/java/org/apereo/cas/support/pac4j/authentication/DelegatedClientFactory.java

```diff
@@ -419,6 +421,19 @@ protected void configureSamlClient(final Collection<IndirectClient> properties)
                 cfg.setWantsAssertionsSigned(saml.isWantsAssertionsSigned());
                 cfg.setUseNameQualifier(saml.isUseNameQualifier());
                 cfg.setAttributeConsumingServiceIndex(saml.getAttributeConsumingServiceIndex());
+                
+                try{
+                    cfg.setSamlMessageStoreFactory(SAMLMessageStoreFactory.class.cast(
+                        Class.forName(saml.getMessageStoreFactory()).getDeclaredConstructor().newInstance()));
+                }catch(ClassNotFoundException 
+                        | InstantiationException 
+                        | IllegalAccessException 
+                        | NoSuchMethodException 
+                        | InvocationTargetException e){
+                    LOGGER.error(
```

Comment:
Use `[{}]`

Reply:
Sorry Misagh but I didn't understand.
What do you mean?

Reply:
He just means to put brackets around the braces so it will be obvious if the saml.getMessageStoreFactory() is null. Output would be ```... class [org.xyz.Class]```

Reply:
ah, thank you for clarification